src/components/help/views/ReportSummaryView.tsx

```diff
@@ -35,9 +36,6 @@ export const ReportSummaryView: FC<{}> = props =>
             case ReportType.MESSAGE:
                 SendMessageComposer(new CallForHelpFromForumMessageMessageComposer(activeReport.groupId, activeReport.threadId, activeReport.messageId, activeReport.cfhTopic, activeReport.message));
                 break;
-            case ReportType.PHOTO:
```

Comment:
Why remove this? This is how flash works and I think we should stick to original functionality